### PR TITLE
PHPStan 1.3.3, Psalm 4.18.1

### PR DESCRIPTION
**Pull request by derrabus**

### CODE REVIEW

File: psalm-baseline.xml [hunk 1]
```diff
@@ -2192,10 +2191,6 @@
     </PropertyNotSetInConstructor>
   </file>
   <file src="lib/Doctrine/ORM/Query/AST/Functions/SubstringFunction.php">
-    <InvalidScalarArgument occurrences="2">
-      <code>$optionalSecondSimpleArithmeticExpression</code>
-      <code>$sqlWalker-&gt;walkSimpleArithmeticExpression($this-&gt;firstSimpleArithmeticExpression)</code>
-    </InvalidScalarArgument>
```
Comment (greg0ire) [inline psalm-baseline.xml]: Some of these would need to be "moved" to a baseline specific to 2.13

Comment (greg0ire) [reply psalm-baseline.xml]: See https://github.com/doctrine/orm/pull/9343

**Comment by greg0ire** [comment]
Sorry, I meant to close my own PR…

File: psalm-baseline.xml (hunk 1 above)
Comment (derrabus) [reply psalm-baseline.xml]: I could also move them to psalm.xml.dist and add a comment that this applies to DBAL 2 only. That should stabilize the baseline as well.

**Review by greg0ire — APPROVED** [review]

**Pull request merged**